src/Forms/src/Xamarin.Forms.Core/Button.cs

```diff
@@ -383,31 +383,6 @@ void ITextElement.OnTextTransformChanged(TextTransform oldValue, TextTransform n
 		public virtual string UpdateFormsText(string source, TextTransform textTransform)
 			=> TextTransformUtilites.GetTransformedText(source, textTransform);
 
-		[DebuggerDisplay("Image Position = {Position}, Spacing = {Spacing}")]
-		[TypeConverter(typeof(ButtonContentTypeConverter))]
-		public sealed class ButtonContentLayout
```

Comment:
Are we actively using this class on this PR? If not can we just move this back here?